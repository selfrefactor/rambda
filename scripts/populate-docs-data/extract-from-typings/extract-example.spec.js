import { extractExample } from './extract-example'

test('happy', () => {
  expect(extractExample()).toMatchSnapshot()
})
