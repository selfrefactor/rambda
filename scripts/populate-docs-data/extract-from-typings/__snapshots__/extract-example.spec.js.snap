// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`happy 1`] = `
Object {
  "add": "R.add(2, 3) // =>  5",
  "adjust": "R.adjust(
  0,
  a => a + 1,
  [0, 100]
) // => [1, 100]",
  "all": "const list = [ 0, 1, 2, 3, 4 ]
const predicate = x => x > -1

const result = R.all(fn, arr)
// => true",
}
`;
